README.md

```diff
@@ -1,68 +1,132 @@
 
-## 📦 What’s in the box?
+**Typescript support**
 
-- [File uploading solutions](/solutions/file-uploader/) — file uploader implementations for the most frequent use cases.
-- [Blocks to build custom solutions](/blocks/) — build your integrations with a simple HTML and CSS.
-- [Adaptive image](/solutions/adaptive-image/) — all you need to distribute your web images efficiently.
-- [Cloud image editor](/solutions/cloud-image-editor/) — tune your pictures with our CDN image processing right after they're uploaded.
-- Soon: web components for media content representation: audio, video, image galleries, etc.
+We use [JSDoc type annotations](https://www.typescriptlang.org/docs/handbook/intro-to-js-ts.html) for TypeScript static analysis support during development.  Additionally, we provide type definitions ([\*.d.ts files](https://www.typescriptlang.org/docs/handbook/declaration-files/dts-from-js.html)) for the TypeScript projects in our packages. Check the [JSDoc Reference](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html) page in TypeScript official documentation.
 
-## ⚙️ Installation and start
+## 🚀 Getting Started
 
-Installation process is easy and takes minutes. See the [installation guide](/get-started/installation/).
+### HTML `<script>` Tag
 
-## Typescript support
+* Connect `Blocks` directly from your document:
 
-We use [JSDoc type annotations](https://www.typescriptlang.org/docs/handbook/intro-to-js-ts.html) for TypeScript static analysis support during development.
+```html
+<script src="https://unpkg.com/@uploadcare/blocks@{{PACKAGE_VERSION}}/web/blocks-browser.min.js" type="module"></script>
+```
 
-Additionally, we provide type definitions ([\*.d.ts files](https://www.typescriptlang.org/docs/handbook/declaration-files/dts-from-js.html)) for the TypeScript projects in our packages.
+* Start using `Blocks` in your application markup:
 
-Check the [JSDoc Reference](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html) page in TypeScript official documentation.
+```html
+<lr-file-uploader-regular
+  css-src="https://unpkg.com/@uploadcare/blocks@{{PACKAGE_VERSION}}/web/file-uploader-regular.min.css"
+>
+</lr-file-uploader-regular>
+```
 
-## Supported browsers
+### Using CLI
 
-Blocks are supported and tested in all major modern desktop and mobile browsers:
+You’ll need [Node.js](https://nodejs.org/) installed on your computer.
 
-- Chrome
-- Firefox
-- Safari
-- Edge
-- Opera
+* Install Blocks package: `npm i --save-exact @uploadcare/blocks`
+* Use `connectBlocksFrom` function to connect blocks:
+```js
+import { connectBlocksFrom } from '@uploadcare/blocks/abstract/connectBlocksFrom.js';
```

Comment:
There should be the same HTML snippet and the following js snippet:

```js
  import * as LR from '@uploadcare/blocks';
  LR.registerBlocks(LR);
  ```
  
`connectBlocksFrom` is dirty thing